docs/doc/10-deploy/00-understanding-deployment-modes.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Understanding Deployment Modes
+sidebar_label: Understanding Deployment Modes
+description:
+  Describes Databend deployment modes
+---
+
+## Understanding Deployment Modes
+
+A Databend deployment includes two types of node, Meta and Query. The Meta node stores various types of metadata (such as database, table, cluster, and transaction) and manages user information (including authorization and authentication). The Query node takes care of queries. 
+
+When deploying Databend, you specify a deployment mode, standalone, or cluster. A standalone Databend allows for one Meta node and one Query node, and a Databend cluster can include multiple Meta and Query nodes to enhance the computing capability.
+
+### Compatible Object Storage Solutions
+Databend works with Amazon S3 compatible self-hosted or cloud object storage solutions. You will need to prepare your own object storage before deploying Databend. The following is a list of compatible object storage solutions:
+- Self-hosted object storage solutions:
+  - MinIO
+  - SeaweedFS
+- Cloud object storage solutions:
+  - Amazon S3
+  - Tencent COS
+  - Alibaba OSS
+  - Wasabi
+  - QingCloud QingStore
+  - Azure Blob Storage
```

Comment:
Azblob is a separate storage service that is not compatible with AWS s3.

It's better to split the layout to:

- s3
  - aws s3
  - cos
  - oss
  - ...
- azblob

Reply:
Also, I don't like the idea to mention s3 everywhere (and in our images).

Databend can be deployed on **any** storage services, s3 is one of them. A.k.a, databend is vendor-neutral.